Abstract/portable-formula.rb

```diff
@@ -48,27 +48,21 @@ def self.inherited(subclass)
     subclass.class_eval do
       keg_only "portable formulae are keg-only"
 
-      # TODO remove `subclass.name !~ /PortableRuby$/` when updating portable-ruby to 2.1 or above.
-      option "without-universal", "Don't build a universal binary" if OS.mac? && subclass.name !~ /PortableRuby$/
+      option "without-universal", "Don't build a universal binary"
```

Comment:
To prevent running `lipo` on Linux, which are guarded by `if build.with? "universal"`, change this line to:
```ruby
option "without-universal", "Don't build a universal binary" if OS.mac?
```